### PR TITLE
Youtube media publishers now show properly in rewards panel

### LINKED ISSUE #116: Add support for media provider in the panel

### PR DESCRIPTION
**Pull request by jasonrsadler**
Fixes #116 

core implementation https://github.com/brave/brave-core/pull/699

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-.idea
\ No newline at end of file
+.idea
+.vscode
```
Comment (ryanml) [inline .gitignore]: Did you mean to add this?

Comment (jasonrsadler) [reply .gitignore]: Yes, `.vscode` directory should never be checked in.

Comment (jasonrsadler) [reply .gitignore]: Yes. `.vscode` directory should never be checked in.

Comment (jasonrsadler) [reply .gitignore]: Yes. `.vscode` directory should never be checked in.

Comment (jasonrsadler) [reply .gitignore]: Yes. `.vscode` directory should never be included.

Comment (jasonrsadler) [reply .gitignore]: Yes. `.vscode` directory should never be included.

Comment (jasonrsadler) [reply .gitignore]: Yes, `.vscode` directory should not be checked in.

**Comment by jasonrsadler** [comment]
I am seeing some problems with certain custom paths for some publishers not showing up. Created an issue in https://github.com/brave/brave-browser/issues/1886

**Pull request merged**